jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -55,8 +55,16 @@ BREW_BASE_DEPENDCIES="mercurial git cmake"
 # 1. SDFORMAT
 # ruby for xml_schemas generation and libxml2-utils for xmllint used in tests
 SDFORMAT_NO_IGN_DEPENDENCIES="${pythonv}     \\
-                              libxml2-utils  \\
-                              libtinyxml-dev"
+                              libxml2-utils"
+
+# SDFORMAT 10 and above use tinyxml2
+if [[ ${SDFORMAT_MAJOR_VERSION} -ge 10]]; then
+  SDFORMAT_NO_IGN_DEPENDENCIES="${SDFORMAT_NO_IGN_DEPENDENCIES} \\
+                                libtinyxml2-dev"
```

Comment:
this is what it should look like eventually, but until everything is merged, I think it will keep other branches targeting `sdf10` working if we install both `libtinyxml-dev` and `libtinyxml2-dev` for 10+. Once https://github.com/osrf/sdformat/pull/264 is merged, the `libtinyxml-dev` line can be removed.

how does that sound?

Reply:
Completely reasonable, updated.